core/contracts/oracle/implementation/Voting.sol

```diff
@@ -747,4 +742,8 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface, Encrypte
             return RequestStatus.Future;
         }
     }
+
+    function _getIdentifierWhitelist() private view returns (IdentifierWhitelistInterface supportedIdentifiers) {
+        return IdentifierWhitelistInterface(finder.getImplementationAddress("IdentifierWhitelist"));
```

Comment:
This looks great! you'll need to refactor this part once the [this](https://github.com/UMAprotocol/protocol/pull/1241) PR is merged unfortunately though. I'll merge it in as soon as CI passes.

Reply:
sounds good